contentreview-impl/impl/src/java/org/sakaiproject/contentreview/impl/turnitin/TurnitinReviewServiceImpl.java

```diff
@@ -906,6 +1012,222 @@ public void addTurnitinInstructor(Map userparams) throws SubmissionException, Tr
 	@SuppressWarnings("unchecked")
 	public void createAssignment(String siteId, String taskId, Map extraAsnnOpts) throws SubmissionException, TransientSubmissionException {
 
+		Site s = null;
+		try {
+			s = siteService.getSite(siteId);
+		} catch (IdUnusedException iue) {
+			log.warn("createAssignment: Site " + siteId + " not found!" + iue.getMessage());
+			throw new TransientSubmissionException("Create Assignment not successful. Site " + siteId + " not found");
+		}
+		
+		//////////////////////////////  NEW LTI INTEGRATION  ///////////////////////////////
```

Comment:
Most of it belongs to the old integration, which should be deleted in the future. Maybe we can extract all that to a different method?